usermods/smartnest/usermod_smartnest.h

```diff
@@ -168,4 +172,34 @@ class Smartnest : public Usermod
   {
     return USERMOD_ID_SMARTNEST;
   }
+
+  /**
+   * setup() is called once at startup to initialize the usermod.
+   */
+  void setup() {
+      DEBUG_PRINTF("Smartnest usermod setup initializing...");
+      
+      // Publish initial status
+      sendToBroker("report/status", "Smartnest usermod initialized");
```

Comment:
I know this is a bit late but I have figured out that WiFi is not yet configured let alone connected at the time of `setup()` call.
This call is pointless.